ts-defs/index.d.ts

```diff
@@ -1861,15 +1861,17 @@ interface FixtureFn {
      * Declares a test fixture.
      *
      * @param name - The name of the fixture.
+     * @param tagArgs - tag literal arguments required to support the "fixture`${x}`" syntax
```

Comment:
AWAIK, if we're talking about the whole ``fn `str` ``  literal - it's a *tagged literal*. If we're talking about `fn` - it's a *tag function* or *tag expression*.
Thus, I'm not sure about `tag literal`.

Reply:
You're right, we are talking about `fn` here, so it must be `tag function arguments`?

Reply:
Yes, seems like it should be `tag function arguments`